app/config/FrontendAppConfig.scala

```diff
@@ -58,4 +58,9 @@ class FrontendAppConfig @Inject() (config: Configuration, servicesConfig: MyServ
 
   val encryptionKey: String      = config.get[String]("encryption.key")
   val encryptionEnabled: Boolean = config.get[Boolean]("encryption.enabled")
+
+  lazy val manageTransitMovementsUrl: String                = config.get[String]("urls.manageTransitMovementsFrontend")
+  lazy val manageTransitMovementsViewDeparturesUrl: String  = s"$manageTransitMovementsUrl/view-departure-declarations"
+  lazy val manageTransitMakeDepartureDeclarationUrl: String = config.get[String]("urls.manageTransitMovementsDepartureDeclaration")
+  lazy val urlForTesting                                    = s"$manageTransitMakeDepartureDeclarationUrl/local-reference-number"
```

Comment:
manage-transit-movements/departures redirects to the LRN page so we can remove this and use the above val instead